### ISSUE TITLE
Update Hungarian translations

### ISSUE THREAD

**Issue by tibitoth** [squidfunk/mkdocs-material#4967]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "hu",
  "action.edit": "Oldal szerkesztése",
  "action.skip": "Kihagyás",
  "action.view": "Oldal forrásának megtekintése",
  "announce.dismiss": "Ne mutasd többet",
  "blog.archive": "Archívum",
  "blog.categories": "Kategóriák",
  "blog.categories.in": "kategória:",
  "blog.continue": "Folytatás",
  "blog.draft": "Piszkozat",
  "blog.index": "Vissza a főoldalra",
  "blog.meta": "Metaadat",
  "blog.references": "Kapcsolódó linkek",
  "clipboard.copy": "Másolás vágólapra",
  "clipboard.copied": "Vágólapra másolva",
  "consent.accept": "Elfogadás",
  "consent.manage": "Beállítások",
  "consent.reject": "Visszautasítás",
  "footer": "Élőláb",
  "footer.next": "Következő",
  "footer.previous": "Előző",
  "header": "Élőfej",
  "meta.comments": "Hozzászólások",
  "meta.source": "Forrás",
  "nav": "Navigáció",
  "readtime.one": "1 percnyi",
  "readtime.other": "# percnyi",
  "rss.created": "RSS feed",
  "rss.updated": "Frissített tartalom RSS feedje",
  "search": "Keresés",
  "search.placeholder": "Keresés",
  "search.share": "Megosztás",
  "search.reset": "Törlés",
  "search.result.initializer": "Keresés inicializálása",
  "search.result.placeholder": "Kereséshez írj ide valamit",
  "search.result.none": "Nincs találat",
  "search.result.one": "1 egyező dokumentum",
  "search.result.other": "# egyező dokumentum",
  "search.result.more.one": "1 további találat az oldalon",
  "search.result.more.other": "# további találat az oldalon",
  "search.result.term.missing": "Üres",
  "select.language": "Nyelvváltás",
  "select.version": "Verzióváltás",
  "source": "Főoldalra ugrás",
  "source.file.contributors": "Szerzők",
  "source.file.date.created": "Létrehozva",
  "source.file.date.updated": "Utolsó frissítés",
  "tabs": "Lapok",
  "toc": "Tartalomjegyzék",
  "top": "Vissza a tetejére"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to my best knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by tibitoth** [comment]
@squidfunk only one change:   `"blog.categories.in": "kategória:",`

Hard to translate the `in` keyword becuase in hungarian we use suffixes for this kind of relationship e.g.: in Search ==> Keresés**ben**, in Blog ==> Blog**ban**. So instead of mimicing this kind of inclusion I simply declared this as a label: in Search ==> kategória: Search which means category: search

**Comment by squidfunk** [comment]
Thanks! Shouldn't `kategória` be capitalized?

**Comment by tibitoth** [comment]
Yeah it should be

**Comment by squidfunk** [comment]
Perfect, thanks! Added in https://github.com/squidfunk/mkdocs-material/commit/0c9fd53df409fe523de85309886649f384912518.

**Comment by kamilkrzyskow** [comment]
> squidfunk only one change: `"blog.categories.in": "kategória:",`
> 
> Hard to translate the `in` keyword becuase in hungarian we use suffixes for this kind of relationship e.g.: in Search ==> Keresés**ben**, in Blog ==> Blog**ban**. So instead of mimicing this kind of inclusion I simply declared this as a label: e.g.: in Search ==> kategória: Search which means category: search

There was a similar issue in Polish, we also couldn't use a simple `in` equivalent, because it depends on the context. It's been decided to keep it empty. The idea with the `Category: ` suffix also seems good, but then again https://squidfunk.github.io/mkdocs-material/blog/category/performance/ has 2 categories, which then would not fit quite well with a singular `Category: `, at least in Polish. So I still think that leaving it empty is better for Polish. Just food for thought @tibitoth.

**Comment by tibitoth** [comment]
They're both (singular and plural) sounds good in hungarian, we could also leave it empty. I think just leave it in the current state.

**Comment by squidfunk** [comment]
Released as part of [9.0.10](https://github.com/squidfunk/mkdocs-material/releases/tag/9.0.10).
